package.json

```diff
@@ -27,6 +27,7 @@
     "@trivago/prettier-plugin-sort-imports": "^4.1.1",
     "@types/chai": "^4.3.4",
     "@types/dompurify": "^2.3.3",
+    "@types/enzyme": "^3.10.12",
```

Comment:
The `@types/enzyme` package brings in the React typings, which I think is likely to lead to problems down the line.  I'm not aware of an existing package that provides Enzyme types with Preact. It would be useful if the `enzyme-adapter-preact-pure` package provided them. One possibility would be to fork the [Enzyme types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/enzyme/index.d.ts) and replace the references the React types with Preact ones. I can talk to the other Preact developers in case they have other ideas.

Reply:
Yeah, I foresaw a problem with this as well, however it did not fail "too much" except for this: https://github.com/preactjs/preact/issues/1206#issuecomment-854862736

I don't know if it could have other side effects as we adopted TS further. Maybe I can try migrating a couple more components just in case, or at least try to analyze what we import from enzyme and how we use it.

Reply:
I think I prematurely added `@types/enzyme` assuming it would be needed, but I didn't check if the enzyme types provided `enzyme-adapter-preact-pure` would be enough.

If it is, that would solve the problems and side effects caused by bringing React types to the project.